indico/modules/rb_new/schemas.py

```diff
@@ -266,8 +284,9 @@ class Meta:
 rooms_schema = RoomSchema(many=True)
 room_attribute_values_schema = RoomAttributeValuesSchema(many=True)
 map_areas_schema = MapAreaSchema(many=True)
-reservation_occurrences_schema = ReservationOccurrenceSchema(many=True)
-reservation_schema = ReservationSchema()
+reservation_occurrences_min_schema = ReservationOccurrenceMinSchema(many=True)
+reservation_occurrences_full_schema = ReservationOccurrenceFullSchema(many=True)
```

Comment:
just thinking, should we start to get rid of `many=True` when instantiating schemas and instead pass it when serializing? right now we have an unholy mix of both and then sometimes setting/unsetting it at serialization time

Reply:
Yep. I noticed that too.